lib/avo/configuration.rb

```diff
@@ -44,6 +44,7 @@ class Configuration
     attr_accessor :sign_out_path_name
     attr_accessor :resources
     attr_accessor :prefix_path
+    attr_accessor :parent_controller
```

Comment:
Maybe this should be `parent_resource_controller`? Who knows what kind of other controllers we might introduce in the future.

Reply:
We have an controller generator not an resource_controller generator, `rails g avo:controller city --parent-controller SomeRandomController`.

I agree with your suggestion but should we change the above generator to: `rails g avo:resource_controller city --parent-resource-controller SomeRandomController` ?

Reply:
`resource_parent_controller`